src/js/labels/labels.js

```diff
@@ -18,6 +18,8 @@ export const verbs = {
   MUTE: () => 'mute',
   UNMUTE: () => 'unmute',
   AIRPLAY: () => 'air play',
+  ENTER_CAST: () => 'enter cast mode',
```

Comment:
Similar comment here. Is "enter cast" the right phrasing? Something like "start casting" sounds a little better to me.

Reply:
renamed to start / stop casting